test/e2e/util/devfile-example.yaml

```diff
@@ -0,0 +1,67 @@
+metadata:
+  name: quarkus-tests
+projects:
+  - name: quarkus-quickstarts
```

Comment:
Use the very basic devfile.
No plugins, no commands

Reply:
ok